src/lib/data/audio.ts

```diff
@@ -147,6 +147,10 @@ function updateTime() {
 }
 // calls updateTime() every 100ms
 function toggleTimeRunning() {
+    // enables audio play position to be changed by clicking on verse number
+    if (currentAudioPlayer.isAudio === false) {
```

Comment:
It is not good to add additional state which can get out-of-sync.  Could you have a method call hasAudioPlayed which checks the currentAudioPlayer.progress > 0?

Reply:
If we did keep it as a state variable, "isAudio" is not the best name.